### ISSUE TITLE
Add option to get_* to print API call?

### ISSUE THREAD

**Issue by mfherman** [walkerke/tidycensus#194]
I was thinking it could be handy for debugging purposes to have an option for `get_acs` and friends to print the url of the API call that it is using. It could enable a user to copy and paste that call into a browser and help to determine if the issue is with tidycensus or the Census API.

It could look something like this:

``` r
get_acs(
  geography = "county",
  state = "VT",
  variables = "B01003_001",
  show_call = TRUE
  )
#> Getting data from the 2013-2017 5-year ACS
#> Census API call: https://api.census.gov/data/2017/acs/acs5?get=B01003_001E%2CB01003_001M%2CNAME&for=county%3A%2A&in=state%3A50
#> # A tibble: 14 x 5
#>    GEOID NAME                       variable   estimate   moe
#>    <chr> <chr>                      <chr>         <dbl> <dbl>
#>  1 50001 Addison County, Vermont    B01003_001    36825    NA
#>  2 50003 Bennington County, Vermont B01003_001    36054    NA
#>  3 50005 Caledonia County, Vermont  B01003_001    30576    NA
#>  4 50007 Chittenden County, Vermont B01003_001   160985    NA
#>  5 50009 Essex County, Vermont      B01003_001     6203    NA
#>  6 50011 Franklin County, Vermont   B01003_001    48816    NA
#>  7 50013 Grand Isle County, Vermont B01003_001     6950    NA
#>  8 50015 Lamoille County, Vermont   B01003_001    25191    NA
#>  9 50017 Orange County, Vermont     B01003_001    28901    NA
#> 10 50019 Orleans County, Vermont    B01003_001    26951    NA
#> 11 50021 Rutland County, Vermont    B01003_001    59676    NA
#> 12 50023 Washington County, Vermont B01003_001    58691    NA
#> 13 50025 Windham County, Vermont    B01003_001    43332    NA
#> 14 50027 Windsor County, Vermont    B01003_001    55485    NA
```

<sup>Created on 2019-10-01 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

@walkerke, let me know what you think and I can implement for all the `get_*()` functions and figure out how to display when we're making multiple calls and combining the results.

For the example above, I just added the following to `load_data_acs()`

```
  if (show_call) {
    call_url <- gsub("&key.*", "", call$url)
    message(paste("Census API call:", call_url))
  }
```

**Comment by plnnr** [comment]
This would be a really handy feature! For multiple calls, just print all the API calls at the start, maybe with some kind of `suppress_call_url` parameter.

**Comment by mfherman** [comment]
This is now implemented in the dev version of tidycensus via #203
